check-code-style/check_style_bzl.sh

```diff
@@ -12,9 +12,10 @@ if [[ $? != 0 ]]; then
   exit 1
 fi
 
-# Find all files we want to check 
+# Find all files we want to check excluding directories with 'submodules' name.
```

Comment:
```suggestion
# Find all files we want to check excluding directories with 'submodules' name.
#
# TODO(artur): take a list of directories to exclude as part of the action so that
# anyone using this can exclude what ever directory they want rather than assume
# that everyone using this will want to exclude the 'submodules' directory. 
```